songs/die-aerzte/die-perfekte-diktatur.tex

```diff
@@ -0,0 +1,96 @@
+\beginsong{Die perfekte Diktatur}[by={Die Ärzte}]
+
+    \textnote{Your guitar should be tuned to an E$\,\flat$}
+    Intro: Am Em Fmaj7 Fadd9 (2x)
+    \beginverse
+        \[Am] Ich geh nicht mehr ans Tele\[Em]fon,
+        Es klingelt jeden Tag um \[Fmaj7]vier. \[Fadd9]
+        \[Am] Ich fahr nicht mit der Strassen\[Em]bahn,
+        Ich lauf den ganzen Weg zu \[Fmaj7]dir. \[Fadd9]
+        \[C] Das Telefon wird abge\[G]hört
+        und in der Bahn sitzen Spi\[Dm]one. \[Dsus]
```

Comment:
Which `Dsus` is meant here?
Please specify with `Dsus2` or `Dsus4`.

Reply:
I'm not sure. I'm thinking the website I pulled the riff from isn't entirely trustworthy.
Rewatching [this video](https://www.youtube.com/watch?v=LB57MEDteck) I only see 
Farin Urlaub playing a `Dm`. Will delete all `Dsus` chords accordingly.